sopel/modules/reddit.py

```diff
@@ -123,43 +124,42 @@ def redditor_info(bot, trigger, match=None):
     match = match or trigger
     try:
         u = r.get_redditor(match.group(2))
```

Comment:
What started this was `get_redditor` being removed in praw 4+, right?

Reply:
Correct. Major function names changed between praw 3 and 4

Reply:
But this patch requires `praw>=4.0.0` while using the `praw<4.0.0` redditor method name…

Reply:
Fixed now